internal/datacoord/compaction_task_clustering.go

```diff
@@ -615,6 +618,10 @@ func (t *clusteringCompactionTask) NeedReAssignNodeID() bool {
 	return t.GetState() == datapb.CompactionTaskState_pipelining && (t.GetNodeID() == 0 || t.GetNodeID() == NullNodeID)
 }
 
+func (t *clusteringCompactionTask) GetSlotUsage() int64 {
+	return t.slotUsage
```

Comment:
maybe we can just return `paramtable.Get().DataCoordCfg.ClusteringCompactionSlotUsage.GetAsInt64()` ?

Reply:
Yes, I initially implemented it as you suggested, but I found it is hard to add UT. Additionally, we may need to adjust slotUsage based on factors like data size or other policies. Even tasks of the same compaction type may require different slotUsage values.

Reply:
Got it.